Travel-Planner/Travel-Planner/Views/ItineraryView/DayPlan/DayPlanTitleAndDescriptionView.swift

```diff
@@ -4,27 +4,27 @@
 //
 //  Created by Jason Zeng on 10/13/24.
 //
-
-import SwiftUI
-
-struct DayPlanTitleAndDescriptionView: View {
-    
-    @Binding var dayPlan: DayPlan
-    
-    var body: some View {
-        HStack{
-            Text("Day \(dayPlan.dayNumber):")
-            TextField("Day Title", text: $dayPlan.title)
-            
-            Spacer()
-        }
-        .font(.title2)
-        .fontWeight(.semibold)
-        
-        
-        TextField("Day Description", text: $dayPlan.description)
-            .font(.body)
-            .padding(.top, 3)
```

Comment:
DayPlan still need a title and description, find a way to implement it. Add a edit button on the navigationbar trailling to edit trip info?